app/controllers/sitemaps_controller.rb

```diff
@@ -1,8 +1,9 @@
 class SitemapsController < ApplicationController
   def show
     @courses = Course
+      .includes(:provider)
       .where(recruitment_cycle_year: Settings.current_cycle)
-      .select('course_code', 'provider_code', 'changed_at')
+      .select('code', 'changed_at')
       .page(1)
       .per(20_000)
       .all
```

Comment:
This no longer works as intended

https://api.qa.publish-teacher-training-courses.service.gov.uk/api-reference.html#schema-pagination

You got about 500 vs 12281


https://qa.find-postgraduate-teacher-training.service.gov.uk//sitemap.xml
https://www.find-postgraduate-teacher-training.service.gov.uk//sitemap.xml